libr/anal/fcn.c

```diff
@@ -1326,11 +1327,8 @@ static int fcn_recurse(RAnal *anal, RAnalFunction *fcn, ut64 addr, int depth) {
 				if (must_eob) {
 					FITFCNSZ ();
 					op.jump = UT64_MAX;
-					//  recurseAt (op.jump);
-					if (op.fail != UT64_MAX) {
-						// recurseAt (op.fail);
-					}
-					gotoBeach (R_ANAL_RET_END);
+					gotoBeachRet ();
+					return R_ANAL_RET_END;
```

Comment:
Lines 1330-1331 are due to rebase after fix (#13387) went into master.

Reply:
the gotoBeachRet is a goto, so the return statement shouldnt affect the result